src/test/java/com/github/fge/jsonpatch/AddOperationTest.java

```diff
@@ -22,11 +22,11 @@
 import java.io.IOException;
 
 public final class AddOperationTest
-    extends JsonPatchOperationTest
+    extends StandardJsonPatchOperationTest
 {
     public AddOperationTest()
         throws IOException
     {
-        super("add");
+        super("add", AddOperation.class);
```

Comment:
I really do think that having an
interface JsonPatchOperation {
  public String getOpKey();
  public Class<? extends JsonPatchOperationImpl> getImpl();
}

Or something would improve accidentally mis-matching impls with op keys, but I'm willing to let it go.